python/features/steps/general_purpose_steps.py

```diff
@@ -42,6 +42,26 @@ def run_cli_command_with_args(command_name: str, args: list[str]) -> Tuple[int,
     return exit_code, output_message
 
 
+def run_cli_command_with_args_with_stdout(command_name: str, args: list[str]) -> Tuple[int, str, str]:
+    """
+    Utility function to invoke the CLI command with the given arguments.
+
+    Args:
+        command_name (str): CLI command to be executed.
+        args (list[str]): List of arguments associated with the CLI command.
+
+    Return:
+        Exit code and output message resulting from the execution of the CLI command.
+    """
+    initialize_cli()
+    runner = CliRunner()
+    result = runner.invoke(cli, [command_name] + args)
+    exit_code = result.exit_code
+    std_out = str(result.stdout)
+    output_message = std_out.strip().replace("\x1b[0m", "")
+    return exit_code, output_message
```

Comment:
If `std_out` isn't actually necessary, a repeat method for running the cli command is not necessary. 

```suggestion
```